kyaml/fn/framework/processors.go

```diff
@@ -0,0 +1,394 @@
+// Copyright 2021 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package framework
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/markbates/pkger"
```

Comment:
I'm not sure it's worth depending on this package for the framework, especially since Go 1.16 provides a better way to embed files.

Reply:
This is already a dependency in the existing version. We could explore removing it later, but that would require reworking helpers. I'd like to consider it out of scope for this PR.